xapian-glib/xapian-term-generator.h

```diff
@@ -60,6 +60,9 @@ void                    xapian_term_generator_set_stemmer               (XapianT
 XAPIAN_GLIB_AVAILABLE_IN_1_0
 void                    xapian_term_generator_set_stemming_strategy     (XapianTermGenerator    *generator,
                                                                          XapianStemStrategy      strategy);
+XAPIAN_GLIB_AVAILABLE_IN_1_2
```

Comment:
1.2 is closed for new API; we'll need to bump up the version to 1.3.1, add the `XAPIAN_GLIB_AVAILABLE_IN_1_4` macros, before merging this PR.